src/docs/references/availability/index.md

```diff
@@ -84,9 +84,15 @@ An exception with start `2018-11-26T00:30.000+01:00` and end
 
 ###### **Example 3:**
 
-Given exception 1 from `2017-11-26T10:00:00.000Z` to
+An exception with start `2018-11-26T00:30.000+01:00` and end
```

Comment:
2018-11-26T00:30.000+01:00 is invalid timestamp, should be 2018-11-26T00:30:00.000+01:00